api/envoy/data/dns/v2alpha/dns_table.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package envoy.data.dns.v2alpha;
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/status.proto";
+
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.data.dns.v2alpha";
+option java_outer_classname = "DnsTableProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+
+// [#protodoc-title: DNS Filter Table Data]
+// :ref:`DNS Filter config overview <config_udp_listener_filters_dns_filter>`.
+
+// This message contains the configuration for the DNS Filter if populated
+// from the control plane
+message DnsTable {
+  // This message contains a list of IP addresses returned for a query for a known name
+  message AddressList {
+    // This field contains a well formed IP address that is returned
+    // in the answer for a name query. The address field can be an
+    // IPv4 or IPv6 address. Address family detection is done automatically
+    // when Envoy parses the string. Since this field is repeated,
+    // Envoy will return one randomly chosen entry from this list in the
+    // DNS response. The random index will vary per query so that we prevent
+    // clients pinning on a single address for a configured domain
+    repeated string address = 1;
```

Comment:
min length 1? Or do you want to allow empty?

Reply:
I'll add validation.   It shouldn't be empty.